README.md

```diff
@@ -679,6 +679,17 @@ Offline Installations   | ✓     | x   | x   | x    |
   they could find ways to do an offline install for any package manager, but only `gomod` supports
   this out of the box (i.e. the user does not need to change their workflow).
 
+### Actual Tool Versions
+
+Tool   | Version |
+---    | ---     |
+Go     | 1.15.12 |
+npm    | 6.14.13 |
+node   | 14.17.0 |
+pip    | 20.2.2  |
+Python | 3.9.5   |
+git    | 2.31.1  |
+
```

Comment:
Could you please add that we support the first version of yarn?

Reply:
Added.